llvm/test/Transforms/LoopVectorize/iv-select-cmp-no-wrap.ll

```diff
@@ -1,8 +1,58 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
```

Comment:
Please, update the checks here in a separate patch and then update it here

Reply:
https://github.com/Mel-Chen/llvm-project/commit/0acc18c6cf803b61d5950a88f964f7b4d82db341
Are you mentioning something like this?

Reply:
@alexey-bataev 
Could you confirm if my understanding is correct so that I can rebase?

Reply:
#117560